### DISCUSSION TITLE
Why the jump forward is disabled when the overlap mode is on?

### DISCUSSION THREAD

**Discussion by WrRan** [sgl-project/sglang#2283] [Q&A]
https://github.com/sgl-project/sglang/blob/883c955489d70a9dbb7c0e5612f8dfc7ced40472/python/sglang/srt/managers/scheduler.py#L178-L179

**Comment by WrRan** [comment]
I couldn't find the reason for this in the relevant issues or commit history. If you have some time, could you please help clarify my question?  @merrymercy

**Comment by merrymercy** [comment]
This has not been supported currently. It requires special handling.
